conf/routes

```diff
@@ -82,6 +86,7 @@ GET     /userapi/labels                         @controllers.UserProfileControll
 GET     /userapi/labelCounts/all                @controllers.UserProfileController.getAllLabelCounts
 GET     /contribution/auditCounts               @controllers.UserProfileController.getAuditCounts
 GET     /contribution/auditCounts/all           @controllers.UserProfileController.getAllAuditCounts
+GET     /userapi/completedMissionCounts/all     @controllers.UserProfileController.getAllUserCompletedMissionCounts
```

Comment:
Yeah, but do you know what the idea is behind the current structure in that "User status" section? As in, do you know what is supposed to be different between the 4 routes before that newline and the routes after it..?